include/devicetree/clocks.h

```diff
@@ -96,6 +129,41 @@ extern "C" {
 #define DT_CLOCKS_CELL_BY_IDX(node_id, cell, idx) \
 	DT_PHA_BY_IDX(node_id, clocks, idx, cell)
 
+/**
+ * @brief Get Clock controller "cell" value by name
+ *
+ * Example devicetree fragment:
+ *
+ *     clk1: clkctrl@... {
+ *             label = "CLK_1";
+ *             #clock-cells = < 2 >;
+ *     };
+ *
+ *     n: node {
+ *             clocks = < &clk1 10 20 >, < &clk1 30 40 >;
+ *             clock-names = "alpha", "beta";
+ *     };
+ *
+ * Bindings fragment for the clk1 node:
+ *
+ *     clock-cells:
+ *       - bus
+ *       - bits
+ *
+ * Example usage:
+ *
+ *     DT_CLOCKS_CELL_BY_NAME(DT_NODELABEL(n), alpha, bus) // 10
+ *     DT_CLOCKS_CELL_BY_NAME(DT_NODELABEL(n), beta, bits) // 40
+ *
+ * @param node_id node identifier
+ * @param cell binding's cell name within the specifier referenced as "name"
+ * @param name lowercase-and-underscores specifier name
+ * @return the value of the cell inside the named specifier by name
```

Comment:
```suggestion
 * @return the value of the cell inside the named specifier
```

Just a bit of redundancy there